custom/icds_reports/management/commands/sql_scripts/fix_past_data_part_1.sql

```diff
@@ -0,0 +1,43 @@
+UPDATE child_health_monthly
+   SET
+      height_measured_in_month = CASE
+        WHEN (recorded_height is not NULL) AND (valid_in_month=1 AND age_tranche::Integer <= 60) THEN 1
+        ELSE
+          0
+        END,
+      current_month_stunting = CASE
```

Comment:
I think calculation would be wrong  for kids who are <6 months because for previous tables these columns  were already not having values for children <6 months.  Eg. In Jan 2019, for kid of 3 months, current_month_stunting was already 0/null because of height eligibility condition applied to it was 6<eligible<=60. Or you are saying height eligibility was not applied to them before?
If no then these calculations needs to updated. 
Example: current_month_stunting is prepared by `zscore_grading_hfa_last_recorded` and `zscore_grading_hfa` which are already stored in the chm table without height eligibility check. Those should be used here.

This is same for all columns below

Reply:
Updated!!